source/04_best_practice/random_seed.rst

```diff
@@ -0,0 +1,76 @@
+Random Seed
+=============
+
+In reinforcement learning, different random numbers as seeds will affect the results of the algorithm. In order to reproduce or fairly compare experimental results of other algorithms, we need to use the same random seeds.
+
+
+First, in each entry function, we have a global random "seed" parameter. For example, in ``ding/entry/serial_entry.py`` ,
+
+.. code:: python
+
+    def serial_pipeline(..., seed: int = 0, ...):
+        ...
+
+In ``ding/utils/default_helper.py`` , we define a ``set_pkg_seed`` function that is called in the entry function (as shown below) to set "seed" of all associated packages used.
+
+
+.. code:: python
+
+    def set_pkg_seed(seed: int, use_cuda: bool = True) -> None:
+        random.seed(seed)
+        np.random.seed(seed)
+        torch.manual_seed(seed)
+        if use_cuda and torch.cuda.is_available():
+            torch.cuda.manual_seed(seed)
+
+For a collector or evaluator environment, if only one seed is given, DI-engine generates a list of random seeds for that set of environments.
+
+.. code:: python
+
+    def seed(self, seed: Union[List[int], int], dynamic_seed: bool = None) -> None:
+        """
+        Overview:
+            Set the seed for each environment.
+        Arguments:
+            - seed (:obj:`Union[List[int], int]`): List of seeds for each environment, \
+                or one seed for the first environment and other seeds are generated automatically.
+        """
+        if isinstance(seed, numbers.Integral):
+            seed = [seed + i for i in range(self.env_num)]
+        elif isinstance(seed, list):
+            assert len(seed) == self._env_num, "len(seed) {:d} != env_num {:d}".format(len(seed), self._env_num)
+            seed = seed
+        self._env_seed = seed
+        self._env_dynamic_seed = dynamic_seed
+
+To make the environment more diverse, DI-engine also allows ``dynamic_seed`` to be enabled when many episodes are run per environment.
```

Comment:
I think we should add an example about the effect of random seed on env settings, maybe LunarLander is a good example [link](https://di-engine-docs.readthedocs.io/zh_CN/latest/13_envs/lunarlander_zh.html#id9)